### PR TITLE
containerapp remove managed environment id

### PR DESCRIPTION
**Pull request by Greedygre**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

az containerapp create
az containerapp update
az containerapp hostname
### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
containerapp

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py [hunk 1]
```diff
@@ -918,3 +919,115 @@ def test_containerapp_scale_revision_copy(self, resource_group):
             JMESPathCheck("properties.template.scale.rules[0].custom.auth[1].secretRef", "app-key"),
 
         ])
+
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="westeurope")
+    def test_containerapp_create_with_yaml(self, resource_group):
+        self.cmd('configure --defaults location={}'.format("westeurope"))
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:926]: self.cmd('configure --defaults location={}'.format(TEST_LOCATION))

Comment (Greedygre) [reply src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py]: Fixed.

**Review by Juliehzl — APPROVED** [review]

**Comment by zhoxing-ms** [comment]
May I ask if this PR change will cause a breaking change for customers?

**Comment by Greedygre** [comment]
@

> May I ask if this PR change will cause a breaking change for customers?

This change will not cause breaking change.
We have done compatibility to ensure that customers' behavior will not be changed.

**Review by howang-ms — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**